errors.go

```diff
@@ -0,0 +1,133 @@
+package da
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	pbda "github.com/rollkit/go-da/types/pb/da"
+)
+
+// Code defines error codes for JSON-RPC.
+//
+// They are reused for gRPC
+type Code int
+
+// Codes are used by JSON-RPC client and server
+const (
+	CodeBlobNotFound               Code = 32001
```

Comment:
just out of curiosity, any specific reasoning behind using 32000s?

Reply:
It satisfies the jsonrpc error spec: https://www.jsonrpc.org/specification#error_object